server/room-battle.ts

```diff
@@ -483,6 +483,7 @@ export class RoomBattle extends RoomGames.RoomGame {
 	p2: RoomBattlePlayer;
 	p3: RoomBattlePlayer;
 	p4: RoomBattlePlayer;
+	inviteOnlySetter: ID | 'staff' | null;
```

Comment:
```suggestion
	inviteOnlySetter: ID | null;
```

(assuming I wasn't wrong)